src/index.html

```diff
@@ -17,62 +17,394 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a  href="#apple" class="nav__link">Apple</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Samsung</a>
+            <a href="#samsung" class="nav__link">Samsung</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Smartphones</a>
+            <a href="#smartphones" class="nav__link">Smartphones</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Laptops & computers</a>
+            <a href="#laptop" class="nav__link">Laptops & computers</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Gadgets</a>
+            <a href="#gadget" class="nav__link">Gadgets</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Tablets</a>
+            <a href="#tablets" class="nav__link">Tablets</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Photo</a>
+            <a href="#photo" class="nav__link">Photo</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Video</a>
+            <a href="#video" class="nav__link">Video</a>
           </li>
         </ul>
       </nav>
     </header>
 
     <main class="catalog">
+      
       <div class="catalog__card">
-        Product card
+        <img 
+            src="./images/imac.jpeg" 
+            alt="Samsung Galaxy S22 2022 8/128GB Green" 
+            class="catalog__img"
+        >
+        <h2 class="catalog_describe">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+    
+        <p class="catalog_product-code">
+          Product code: 195434
+        </p>
+    
+        <div>
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <p class="stars__text"> Reviews:&nbsp5</p>
+          </div>
+        </div>
+    
+        <div class="catalog_price">
+          <p class="catalog_price__text"> 
+            Price:
+          </p>
+    
+          <p class="catalog_price__number">
+            &#36;2,199
+          </p>
+        </div>
+        
+        <a href="#button">
+          <button 
+            class="catalog_button" data-qa="card-hover">
+            Buy
```

Comment:
```suggestion
          <button 
            class="catalog_button"
            data-qa="card-hover"
            >
              Buy
```
And same for other items